pkg/loop/reporting.go

```diff
@@ -19,12 +18,15 @@ type reportingPluginFactoryClient struct {
 	grpc pb.ReportingPluginFactoryClient
 }
 
-func newReportingPluginFactoryClient(lb *lggrBroker, cc *grpc.ClientConn) *reportingPluginFactoryClient {
+func newReportingPluginFactoryClient(lb *lggrBroker, cc grpc.ClientConnInterface) *reportingPluginFactoryClient {
```

Comment:
is a FactoryClient a thing that client to a factory a factory that creates clients?

if the latter, i want it to be renamed to ClientFactory.

Reply:
No it's a client for a [`ReportingPluginFactory`](https://github.com/smartcontractkit/libocr/blob/29f534b2de1a05a91a17aab500aef283c5725956/offchainreporting2/types/types.go#L87).